### DISCUSSION TITLE
Looking to hire - developer to integrate Superfaktúra into Saleor and Dashboard

### DISCUSSION THREAD

**Discussion by djkato** [saleor/saleor#13498] [General]
Hello, we're moving to Saleor, and we'd like someone to create an app/plugin for Saleor, so we can use Superfaktúra - an invoice manager.
their provided docs for REST : [https://github.com/superfaktura/docs](https://github.com/superfaktura/docs)
We would expect this integration to allow us to create and send invoices of orders from Saleor dashboard to Superfaktúra. 
It's a Slovak-centric solution, and any translations necessary for the UI or for the API I will provide.
We aren't a huge company so our budget is limited, so please when reaching out give us an estimated final price for the whole project.

**Comment by flo3-Jason** [comment]
Hi there, we've been working quite a bit with Saleor recently and would be interesting in scoping out your app/plugin with Superfaktura. Understand limited budgets, let's see what we can do. 

Could you drop me over an email at jason@flo3.digital so we can connect?

**Reply by djkato** [reply]
Sent

**Comment by pitkes22** [comment]
Hello 👋,

I've worked on similar projects before, notably integrating Saleor with [Pohoda](https://www.pohoda.sk/). And since I'm from Slovakia, I'm quite familiar with local tools like Superfaktúra and nuances which might be beneficial for this integration.

If you're still looking for someone to help out, feel free to send me an email at padampasam@gmail.com.

**Comment by djkato** [comment]
we already found one :)
